qiskit_optimization/algorithms/multistart_optimizer.py

```diff
@@ -51,8 +51,13 @@ def __init__(self, trials: int = 1, clip: float = 100.0) -> None:
             clip: Clipping parameter for the initial guesses in the multi-start method.
                 If a variable is unbounded then the lower bound and/or upper bound are replaced
                 with the ``-clip`` or ``clip`` values correspondingly for the initial guesses.
+
+        Raises:
+            ValueError: if the variable trials has a value smaller than 1.
         """
         super().__init__()
+        if trials <= 0:
```

Comment:
I guess it would have been simpler and more match the docstring to have had `if trails < 1` in both places rather than <= 0

Reply:
Yes, it makes sense. I update the code and title.